packages/website/src/App.js

```diff
@@ -175,7 +175,7 @@ function App({ element, props: appProps }) {
 }
 
 App.propTypes = {
-  element: PropTypes.elementType,
+  element: PropTypes.node,
```

Comment:
Thoughts on this and the other PropTypes changes?

Reply:
Look better indead!